packages/core/stories/accordion/accordion.stories.tsx

```diff
@@ -186,3 +193,89 @@ export const Status: StoryFn<AccordionGroupProps> = (props) => (
     ))}
   </AccordionGroup>
 );
+
+const accounts = [
+  { name: "Account 1", number: "8736" },
+  { name: "Account 2", number: "2564" },
+];
+
+const features = [
+  {
+    name: "Domestic wires",
+    id: "domestic_wires",
+    description: "Initiate wire transfers to another bank",
+  },
+  {
+    name: "Account transfers",
+    id: "account_transfers",
+    description: "Move money within your accounts",
+  },
+];
+
+export const AdditionalLabels: StoryFn = () => {
+  const [state, setState] = useState<Record<string, string[]>>({
+    domestic_wires: [],
+    account_transfers: [],
+  });
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value;
+    const name = event.target.name;
+
+    setState((prev) => ({
+      ...prev,
+      [name]: prev[name].includes(value)
+        ? prev[name].filter((account) => account !== value)
+        : [...prev[name], value],
+    }));
+  };
+
+  return (
+    <div style={{ width: "80%", height: "100%" }}>
+      <AccordionGroup>
+        {Object.values(features).map(({ name, description, id }) => (
+          <Accordion value={id}>
+            <AccordionHeader>
+              <StackLayout gap={0.5}>
+                <SplitLayout
+                  align="baseline"
+                  startItem={
+                    <Text>
+                      <strong>{name}</strong>
+                    </Text>
+                  }
+                  endItem={
+                    <Label styleAs="label" color="secondary">
```

Comment:
nit, but couldn't we just do...
```suggestion
                    <Label color="secondary">
```